cves/CVE-2018-6621.yml

```diff
@@ -169,10 +178,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    According to https://www.securityfocus.com/bid/102950, the developers 
```

Comment:
The commit message mentions a 'GwanYeong Kim'

Reply:
Thanks. I looked into this and it was true. I missed this detail.